tools/run_tests/performance/scenario_config.py

```diff
@@ -164,7 +172,7 @@ def scenarios(self):
           'benchmark_seconds': BENCHMARK_SECONDS
       }
       yield {
-          'name': 'protobuf_async_qps_unconstrained_%s'
+          'name': 'cpp_generic_async_qps_unconstrained_%s'
```

Comment:
Do you mean the scenario should be changed to actually test with protobufs?
I'm fixing the mismatch between scenario name and actually performing EMPTY_GENERIC_PAYLOAD test here.